api/api-referential/referential-external/src/main/java/fr/gouv/vitamui/referential/external/server/service/archivalprofileunit/ArchivalProfileUnitExternalService.java

```diff
@@ -228,6 +257,18 @@ public ArchivalProfileUnitDto update(ArchivalProfileUnitDto dto, VitamContext vi
         }
     }
 
+    public boolean checkExists(final String criteria) {
+        final VitamContext vitamContext = this.buildVitamContext();
+        // Implémentez la logique dans l'InternalService si nécessaire
+        throw new UnsupportedOperationException("Method not implemented yet in InternalService");
+    }
+
+    public ArchivalProfileUnitDto update(final ArchivalProfileUnitDto dto)
+        throws AccessExternalClientException, InvalidParseOperationException {
+        final VitamContext vitamContext = this.buildVitamContext();
+        return this.update(dto, vitamContext);
+    }
+
```

Comment:
```suggestion
    public ArchivalProfileUnitDto update(final ArchivalProfileUnitDto dto)
        throws AccessExternalClientException, InvalidParseOperationException {
        final VitamContext vitamContext = this.buildVitamContext();
        return this.update(dto, vitamContext);
    }

    public boolean checkExists(final String criteria) {
        final VitamContext vitamContext = this.buildVitamContext();
        // Implémentez la logique dans l'InternalService si nécessaire
        throw new UnsupportedOperationException("Method not implemented yet in InternalService");
    }

```

(bouger update près de l'autre implémentation).